boards/arm/frdm_kw41z/frdm_kw41z.yaml

```diff
@@ -15,3 +15,7 @@ supported:
   - ieee802154
   - spi
   - pwm
+testing:
+  ignore_tags:
+    - bluetooth
+    - net
```

Comment:
Does this really make sense as the board does support 802154?

Reply:
we do not have public link level stack provide, only privde as private repo

Reply:
802.15.4 doesn't require a private repo. We can ignore bluetooth tags, but should keep net.

Reply:
done